lib/cipher_text.rb

```diff
@@ -0,0 +1,11 @@
+class CipherText < Message
+
+  def decrypt(shifts)
+    unshifts = shifts.map{|shift_value| shift_value * -1}
+    decryption = {}
+    decryption[:decryption] = shift_text(unshifts)
+    decryption[:key] = ""
+    decryption[:date] = ""
+    decryption
+  end
```

Comment:
This is the only method unique to CipherText, the rest is inherited from the Message super.